src/components/Filters/Aggregation/AggregationBranch/AggregationBranch.js

```diff
@@ -0,0 +1,145 @@
+import './AggregationBranch.less';
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { FormattedNumber } from 'react-intl';
+import {
+  coalesce,
+  getAllFilters,
+  sanitizeHtmlId,
+  slugify,
+} from '../../../../utils';
+import {
+  removeMultipleFilters,
+  replaceFilters,
+} from '../../../../actions/filter';
+import { selectQueryState } from '../../../../reducers/query/selectors';
+import AggregationItem from '../AggregationItem/AggregationItem';
+import getIcon from '../../../iconMap';
+import { SLUG_SEPARATOR } from '../../../../constants';
+
+export const UNCHECKED = 'UNCHECKED';
+export const INDETERMINATE = 'INDETERMINATE';
+export const CHECKED = 'CHECKED';
+
+const AggregationBranch = ({ fieldName, item, subitems }) => {
```

Comment:
can we make this a named export

`export const AggregationBranch`

and remove the default export

Reply:
This has been updated.